support/SubmitScripts/OceanClang.sh

```diff
@@ -112,23 +112,25 @@ if [[ $checkpoints == 0 ]]; then
   # If a checkpoint is found add one to checkpoint and sumbit next job
   if test -e "${PWD}/SpectreCheckpoint$current_checkpoint"; then
     cp ../OceanClang.sh .
-    sed -i '106 s/0/1/1' OceanClang.sh
+    sed -i 's/checkpoints=0/checkpoints=1/' OceanClang.sh
```

Comment:
Anchor the sed matches at the start of the line (`s/^check`...) so they can't match random non-assignment text.  This one will modify itself, for example (although that doesn't actually break anything).

Do this for the others as well.